README.md

```diff
@@ -0,0 +1,48 @@
+[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=sqale_index)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=code_smells)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=coverage)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Ajava-util-logging-testing&metric=ncloc)](https://sonarcloud.io/dashboard?id=com.exasol%3Ajava-util-logging-testing)
+
+# Overview
+
+The Exasol's testing utilities for `java.util.logging` allow testing log message creation in software that uses the `java.util.logging` framework.
+
+## Utilities
+
+## `CapturingLogHandler`
+
+Add this log handler if you want to record log messages created by your software and check the contents as part of unit tests.
+
+## Customer Support
+
+This is an open source project which is officially supported by Exasol. For any question, you can contact our support team.
+
+## Dependencies
+
+### Run Time Dependencies
+
+Running the Virtual Schema requires a Java Runtime version 9 or later.
+
+### Build Time Dependencies
+
+| Dependency                                                                          | Purpose                                                | License                       |
+|-------------------------------------------------------------------------------------|--------------------------------------------------------|-------------------------------|
+| [Java Hamcrest](http://hamcrest.org/JavaHamcrest/)                                  | Checking for conditions in code via matchers           | BSD License                   |
+| [J5SE](https://github.com/itsallcode/junit5-system-extensions)                      | JUnit5 extensions to test Java System.x functions      | Eclipse Public License 2.0    |
```

Comment:
I think we don't use this dependency in the project. Do we need it for something?

Reply:
Right. I removed the wrong dependency. Should be JUnit5 instead of J5SE.